sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -1812,6 +1830,75 @@ public QueryCompatibilityMode getQueryCompatibilityMode() {
         return changeFeedQueryImpl.executeAsync();
     }
 
+    @Override
+    public Flux<FeedResponse<Document>> readAllDocuments(
```

Comment:
collection stale situation:

delete collection, recreate the collection with the same name.

Reply:
Fixed by moving InvalidPartitionExceptionRetryPolicy one level up